src/_pytest/cacheprovider.py

```diff
@@ -220,12 +220,12 @@ def pytest_make_collect_report(self, collector: nodes.Collector):
             lf_paths = self.lfplugin._last_failed_paths
             res.result = sorted(
                 res.result,
-                key=lambda x: 0 if Path(str(x.fspath)) in lf_paths else 1,
+                key=lambda x: x.path not in lf_paths,
```

Comment:
when readin in the pr i think i should return the if/else as it helps to spot the intent

Reply:
i choose a different way